Crypto/PublicKey+Apple.mm

```diff
@@ -578,11 +578,24 @@ virtual int _sign(int/*mbedtls_md_type_t*/ mbedDigestAlgorithm,
                           "Couldn't create a trust to get certificate chain");
             CFAutorelease(trustRef);
             
+#if TARGET_OS_MACCATALYST
+            if (@available(iOS 12.0, macos 10.14, *)) {
+                CFErrorRef error;
+                BOOL trusted = SecTrustEvaluateWithError(trustRef, &error);
```

Comment:
On line 595, SecTrustEvalue() is call, should SecTrustGetTrustResult be called instead for TARGET_OS_MACCATALYST?

Same comments for similar code down below/